B-array-some/exercise.js

```diff
@@ -8,17 +8,26 @@
 
 var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 
+// var pairsByIndex = [[0, 3], [1, 2], [2, 1], [3, 0]];
 // If there is a null value in the array exit the program with the error code
-// https://nodejs.org/api/process.html#process_process_exit_code
-// process.exit(1);
+// https://nodejs.org/api/process.html#process_process_exit_codeprocess.exit(1);
 
 var students = ["Islam", "Lesley", "Harun", "Rukmini"];
 var mentors = ["Daniel", "Irina", "Mozafar", "Luke"];
 
+var filterarr = [];
+
+pairsByIndex.some(function(item) {
```

Comment:
This technically works but is dangerous. Remember, functions like `some` or `every` are about checking a condition so you wouldn't necessarily expect it to change variables inside as in `forEach`.

The prompt here also wants you to merely check for nulls and then exit out of the program rather than filtering it out as you've done here.